.redhat-ci.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM fedora:24
+FROM fedora:25
 MAINTAINER Jonathan Lebon <jlebon@redhat.com>
```

Comment:
Feel free to remove the `MAINTAINER` line, since I'd rather we all maintain it. :)
I incorrectly thought at the time that this was a required field.